### PR TITLE
Add whitespace for markdown format to work

### LINKED ISSUE #91: Fix formatting of readme
Currently the readme doesn't use markdown formatting correctly.

For example, rather than using

# What Open Source Means To Me

We're using

#What Open Source Means To Me

I believe using correct formatting would improve readability a lot.

I'd like to open a PR to fix this if you wanna take this forward

### PR DESCRIPTION
**Pull request by Roshanjossey**
Added a space after heading formatting syntax like `#` and `###`

Fixes #91

### CODE REVIEW

**Comment by justinclift** [comment]
Visually, this definitely looks better.

In your PR though, it has unneeded trailing white spaces on the end of the heading lines.  Are you ok to remove them?

File: README.md [hunk 1]
```diff
@@ -111,86 +111,86 @@ sharing their solution as well.
...
+### [Randall Leeds](http://bleeds.info/)  
 Open Source means to me that access to literacy is a right rather than a privilege and that even if I choose to sell my labor I never forfeit my right to share the resulting abundance.
 
-###[Adam Lofting](http://adamlofting.com)
+### [Adam Lofting](http://adamlofting.com)  
 Open Source, to me, means using our combined energy to iterate on, rather than replicate, the work of those before us; and enabling those who follow us to do the same.
 
-###[Doug Belshaw](https://dougbelshaw.com/)
-Open source to me has moved from a 'nice to have' (1997 when I first downloaded Red Hat Linux) to something that's essential for trust, transparency, and solidarity (2013/14 post-Snowden era). Like the pull request that created *this* contribution, I want to be able to collaborate, give feedback and discuss the tools that I use in a meaningful way.
+### [Doug Belshaw](https://dougbelshaw.com/)
```
Comment (justinclift) [inline README.md]: Extra whitespaces added here too, after the "Linux)"

File: README.md [hunk 2]
```diff
@@ -215,67 +215,67 @@ It has helped me to realize myself, to realize the fact there are many humble pe
...
 
 Het feit dat open source software 'gratis' is te gebruiken, is in mijn optiek een mooie bijkomstigheid, het feit dat het 'open' en 'vrij' is, vind ik echter veel belangrijker. De hele wereld heeft de mogelijkheid om de software te verbeteren en om mogelijke fouten in de code op te lossen. Op deze manier wordt software commit voor commit een stukje mooier. :smiley:
 
-###[Amit Merchant](http://github.com/amitmerchant1990)
-Open source to me is developing something to your heart's content and publish it to the world with the bottom of your heart :heart:. Open source for me is also a way to make the world a better place just by the things you made(no matter how little they are!) and feeling the immense pleasure of contributing something to the outside world.
+### [Amit Merchant](http://github.com/amitmerchant1990)  
+Open source to me is developing something to your heart's content and publish it to the world with the bottom of your heart :heart:. Open source for me is also a way to make the world a better place just by the things you made(no matter how little they are!)   and feeling the immense pleasure of contributing something to the outside world.
```
Comment (justinclift) [inline README.md]: Also here. :wink:

File: README.md [hunk 3]
```diff
@@ -215,67 +215,67 @@ It has helped me to realize myself, to realize the fact there are many humble pe
...
-###[Mohammad Hossein Mojtahedi](http://twitter.com/mhm5000)
+### [Mohammad Hossein Mojtahedi](http://twitter.com/mhm5000)  
 Open source is like a glass of water to me, which never runs out of water and makes me thirstier each time that I drink from it.
 
 I hope one day I see these, too:
 
 + Open Source Goverment
 + Open Source Law
-+ [Open Source Life](https://github.com/gitlife) ➡ [If anybody wants to help me start this, please... contact me]
++ [Open Source Life](https://github.com/gitlife)   ➡ [If anybody wants to help me start this, please... contact me]
```
Comment (justinclift) [inline README.md]: This one seems correct. :smile:

File: README.md [hunk 4]
```diff
@@ -215,67 +215,67 @@ It has helped me to realize myself, to realize the fact there are many humble pe
...
-###[Patrick Metzdorf](http://github.com/batjko)
+### [Patrick Metzdorf](http://github.com/batjko)  
 Open Source, to me, is a world of learning, of accelerated progress and ever renewed respect and appreciation for mankind's ability to mix creative ingenuity with selfless cooperation.
 
 I have been dipping my toes into Open Source only a few months ago, drawn in by the wider Github community and the incredible work that it generates. And I am already convinced that this is where the future of technological progress lies. In all likelihood also that of education, scientific exploration and application; as well as social organization, in more ways than we can imagine today.
 Undoubtedly, the collective minds of Open Source thinkers will be able to bring its benefits and opportunities to areas of society which we may yet think of as forever closed to such innovation.
 I, for one, can't wait.
 
-###[&! (bitandbang)](https://bnb.cupcake.is/profile)
+### [&! (bitandbang)  ](https://bnb.cupcake.is/profile)
```
Comment (justinclift) [inline README.md]: Extra spaces after the ")" here too.

File: README.md [hunk 5]
```diff
@@ -366,41 +366,41 @@ from another user.
...
 
-*<sub>[from the jargon file definition](http://www.catb.org/jargon/):</sub>*
+*<sub>[from the jargon file definition](http://www.catb.org/jargon/)  :</sub>*
 
 ```c
 deep magic: [poss. from C.S. Lewis's `Narnia' books.] n. An
    awesomely arcane technique central to a program or system, esp. one
    not generally published and available to hackers at large (compare
-   {black art}); one which could only have been composed by a true
+   {black art})  ; one which could only have been composed by a true
```
Comment (justinclift) [inline README.md]: .. also here.

File: README.md [hunk 6]
```diff
@@ -410,29 +410,29 @@ deep magic: [poss. from C.S. Lewis's `Narnia' books.] n. An
...
 
-*<sub>[from wikipedia](http://en.wikipedia.org/wiki/Alchemy):</sub>*
+*<sub>[from wikipedia](http://en.wikipedia.org/wiki/Alchemy)  :</sub>*
 ```bash
 Alchemy is an influential tradition whose practitioners have, from antiquity,
 claimed it to be the precursor to profound powers. The defining objectives of
 alchemy are varied but historically have typically included one or more of the
 following goals: the creation of the fabled 'philosophers stone'; the ability to
-transmute base metals into the noble metals (gold or silver);
+transmute base metals into the noble metals (gold or silver)  ;
```
Comment (justinclift) [inline README.md]: ... here.

File: README.md [hunk 7]
```diff
@@ -111,86 +111,86 @@ sharing their solution as well.
...
+### [William Duyck](http://wduyck.com/)  
 Open source is important. Human collaboration across **an open platform is essential to individual growth and our collective future**. That platform is not just a technical implementation of something, but more an ethos, a way of working together. The open web is the communication tool, open source is the way of working together, and when combined we can play to each other's strengths to better ourselves and others quicker, more effectively, and with higher value.
 
 Open source is not just a licensing agreement, or the ability to look at the code. Its understanding the balance of public vs private. Its the ability to work with others, no matter who they are, where they're from, or what they believe. **Open source is a way of life**.
 
-###[Jim Cresswell](http://twitter.com/JimCresswell)
+### [Jim Cresswell](http://twitter.com/JimCresswell)  
 Learning and creating in new, better ways.
-![new ways](/images/einstein.jpg)
+![new ways](/images/einstein.jpg)
```
Comment (justinclift) [inline README.md]: ... and here.

File: README.md [hunk 8]
```diff
@@ -111,86 +111,86 @@ sharing their solution as well.
...
+### [Christian Heilmann](http://christianheilmann.com)  
+Open Source means to me that you share your creation with the world. You don't just give it out to look at it, you allow people to participate and change the project according to suggestions (after a peer review)  . You build it with the people and for the people. That way open source is not only about technology - it is about collaboration and sharing responsibilities. It means that your product will go on when you are not part of it any more and others can build on what you did. It protects you from burning out on one product and allows others to start with something that already works.
 
-![sharing the dog with the world](/images/dog.jpg "Yes, you may pet me while I wait for my human")
+![sharing the dog with the world](/images/dog.jpg "Yes, you may pet me while I wait for my human")
```
Comment (justinclift) [inline README.md]: ... also here.

**Comment by Roshanjossey** [comment]
@justinclift: I was adding them to have a line bread after them. Just did a batch find and replace.

Lemme check if line break happens without trailing spaces (I was following stackoverflow format)

So many kind of markdown formats

**Comment by justinclift** [comment]
:wink:

**Comment by Roshanjossey** [comment]
Yup confirmed. Trailing spaces are not needed if it is a heading.

I'll quickly remove trailing spaces from all the unnecessary places

File: README.md [hunk 9]
```diff
@@ -215,67 +215,67 @@ It has helped me to realize myself, to realize the fact there are many humble pe
...
-###[Mohammad Hossein Mojtahedi](http://twitter.com/mhm5000)
+### [Mohammad Hossein Mojtahedi](http://twitter.com/mhm5000)
 Open source is like a glass of water to me, which never runs out of water and makes me thirstier each time that I drink from it.
 
 I hope one day I see these, too:
 
 + Open Source Goverment
 + Open Source Law
-+ [Open Source Life](https://github.com/gitlife) ➡ [If anybody wants to help me start this, please... contact me]
++ [Open Source Life](https://github.com/gitlife)➡ [If anybody wants to help me start this, please... contact me]
```
Comment (justinclift) [inline README.md]: Ironically, this is one place where the extra spaces would be useful. :wink:

File: README.md [hunk 10]
```diff
@@ -298,43 +298,43 @@ Nelson Mandela (thanks Mariona, for this one!)
 
 **So forge your own destiny. Choose life. Choose open source <img class="emoji" title=":smiley:" alt=":smiley:" src="https://assets-cdn.github.com/images/icons/emoji/unicode/1f603.png" align="absmiddle" height="20" width="20">**
 
-###[Felipe Moura](http://felipenmoura.org/home)
+### [Felipe Moura](http://felipenmoura.org/home)
 Aprendizagem, e evolução!
 
-A comunicação(por fala, desenhos ou escrita) foi um marco na história da Humanidade, fazendo com que lições aprendidas e novas descobertas pudessem ser passadas para outras pessoas por gerações, e regiões diferentes!
+A comunicação(por fala, desenhos ou escrita)foi um marco na história da Humanidade, fazendo com que lições aprendidas e novas descobertas pudessem ser passadas para outras pessoas por gerações, e regiões diferentes!
```
Comment (justinclift) [inline README.md]: Spaces completely missing here now.

File: README.md [hunk 11]
```diff
@@ -111,86 +111,86 @@ sharing their solution as well.
...
+### [Lukas Blakk](http://lukasblakk.com)
+To me Open Source is a tool in the activist toolbox.  It's how we can get things done without having to wait and ask permission, without having to build every. single. thing. we want before we can accomplish great things. We build on each other's work, we build up each other's work together.  The fact that I could "view source" on web pages in the earlier days of the web was how I taught myself to make my own web pages.  Multiply that out times every thing we do with technology in our lives.  Wherever there is a black box, where a corporate power whose only interest is to continue to funnel money into the pockets of the 1%, we will open it up. We (the open source hackers)will reverse engineer, offer alternatives, show it can be done collectively for greater good than any capitalist enterprise.  Look around you - open source is running so many of the things we take for granted.  It's literally for *everyone* and we'll keep getting more and more people rallying to it, learning it & building it, so that there's always a non-silo option and transparency into the increasingly present technology in our lives.
```
Comment (justinclift) [inline README.md]: Spaces completely missing after the "hackers)" too.

**Comment by justinclift** [comment]
Errr, are you ok to go through and double check the change?  It now has spaces missing that shouldn't be, and there's one spot where the double spaces turned out to be useful. :smile:

**Comment by Roshanjossey** [comment]
@justinclift I didn't understand where double spaces turned out to be useful. Could you gimme a line number?

I'll fix the rest in a second

**Comment by justinclift** [comment]
> Could you gimme a line number?

Line 238, but you've fixed it already. :smile:

I'll test your PR out here again in a minute.

**Comment by justinclift** [comment]
Hmmm, looking at line number 238, the extra spaces didn't really seem to do anything.  Weird.

File: README.md [hunk 12]
```diff
@@ -111,86 +111,86 @@ sharing their solution as well.
...
 
-###[Adam Lofting](http://adamlofting.com)
+### [Adam Lofting](http://adamlofting.com)
 Open Source, to me, means using our combined energy to iterate on, rather than replicate, the work of those before us; and enabling those who follow us to do the same.
 
-###[Doug Belshaw](https://dougbelshaw.com/)
-Open source to me has moved from a 'nice to have' (1997 when I first downloaded Red Hat Linux) to something that's essential for trust, transparency, and solidarity (2013/14 post-Snowden era). Like the pull request that created *this* contribution, I want to be able to collaborate, give feedback and discuss the tools that I use in a meaningful way.
+### [Doug Belshaw](https://dougbelshaw.com/)
+Open source to me has moved from a 'nice to have' (1997 when I first downloaded Red Hat Linux)to something that's essential for trust, transparency, and solidarity (2013/14 post-Snowden era). Like the pull request that created *this* contribution, I want to be able to collaborate, give feedback and discuss the tools that I use in a meaningful way.
```
Comment (justinclift) [inline README.md]: Still missing a space here.  After "Linux)".

File: README.md [hunk 13]
```diff
@@ -215,20 +215,20 @@ It has helped me to realize myself, to realize the fact there are many humble pe
...
 
 Het feit dat open source software 'gratis' is te gebruiken, is in mijn optiek een mooie bijkomstigheid, het feit dat het 'open' en 'vrij' is, vind ik echter veel belangrijker. De hele wereld heeft de mogelijkheid om de software te verbeteren en om mogelijke fouten in de code op te lossen. Op deze manier wordt software commit voor commit een stukje mooier. :smiley:
 
-###[Amit Merchant](http://github.com/amitmerchant1990)
-Open source to me is developing something to your heart's content and publish it to the world with the bottom of your heart :heart:. Open source for me is also a way to make the world a better place just by the things you made(no matter how little they are!) and feeling the immense pleasure of contributing something to the outside world.
+### [Amit Merchant](http://github.com/amitmerchant1990)
+Open source to me is developing something to your heart's content and publish it to the world with the bottom of your heart :heart:. Open source for me is also a way to make the world a better place just by the things you made(no matter how little they are!)and feeling the immense pleasure of contributing something to the outside world.
```
Comment (justinclift) [inline README.md]: "are!)and"

**Comment by Roshanjossey** [comment]
@justinclift: I went through the readme again. I think I've covered everything now.

But please don't take my word for it 😉 . Do check everything once again.

**Comment by justinclift** [comment]
Thanks @Roshanjossey, it's good now and definitely gets it looking back the way it should be. :smile:

**Comment by Roshanjossey** [comment]
@justinclift, I liked your quote here

> Open source means making things easier for the next person, who is trying to solve the same problem I just had.

I can relate to it very much. One of the problems I faced a while back was getting started with contributing to open source.

I've started a project to solve this problem for the next person.

https://github.com/Roshanjossey/first-contributions

It'd be awesome if you could take a look at it and give me feedback

**Comment by justinclift** [comment]
Looking through that, it has a decent workflow for new users (already familiar with command line usage obviously).

I could have sworn the GitHub help section used to have similar information for new users, but looking just now I wasn't able to find it.

No real suggestions spring to mind, apart from being uncertain about the `git rebase upstream/master` line.  It's not uncommon for the rebase command to do "unexpected" stuff, especially when working with repos that's have been screwing around :wink: with.  But, not sure if that's really a concern for the guide.

**Comment by Roshanjossey** [comment]
The guide is concentrated at pull request workflow. Give a hands-on tutorial for new people.

Rest of it (keeping fork synched, project suggestions etc) are acting like additional material and showing paths to go forward. We also have a slack team to support/ mentor people to contribute to other projects.

I also wanna build a web app with better listing of projects, filtering and even notifications.

The big picture is we're not stopping at teaching people how to contribute. We wanna go the whole way of making people consistent contributors

**Comment by nickdesaulniers** [comment]
great, thanks Roshan! 👏

**Comment by Roshanjossey** [comment]
@nickdesaulniers I'd love to hear your thoughts on https://github.com/Roshanjossey/first-contributions as well.

As a person who has spent significant time with the community and care about open source, your feedback will be most useful.

That project is the outcome of all the times I asked myself `what open source means to me?`

**Comment by nickdesaulniers** [comment]
very concise; which is hard to do. great work 🌯

**Pull request merged**